docs/lib/datastore/fragments/js/getting-started/30_platformIntegration.md

```diff
@@ -0,0 +1,25 @@
+The fastest way to get started is using the `amplify-app` npx script.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/wH-UnQy1ltM">
+</iframe>
+<br/>
+
+#### React
```

Comment:
I wonder if we can use one of those framework toggles here to clean it up a bit.

Reply:
hhmm good call, does that video focus on ReactJS?

Reply:
I'm thinking more for the distinction between React and React Native

https://docs.amplify.aws/lib/auth/social/q/platform/js